onnxscript/ir/_enums.py

```diff
@@ -82,6 +82,18 @@ def numpy(self) -> np.dtype:
 
         return onnx.helper.tensor_dtype_to_np_dtype(self)
 
+    @classmethod
+    def from_numpy(cls, dtype: np.dtype) -> DataType:
```

Comment:
Prob bad idea?

Reply:
Because of adding dependency on onnx.helper?

Reply:
Right. But it’s ok I guess?